.gitignore

```diff
@@ -4,6 +4,7 @@ out/
 .DS_Store
 .terraform
 *for_tests.yaml
+.idea/
```

Comment:
it's fine, but you can add this common thing into your global .gitignore of your PC, not the project unless this is what this repo generates

Reply:
true maybe good to keep it just to be safe for other devs as well since its quite common